services/README.adoc

```diff
@@ -176,6 +176,13 @@ This includes:
 .. Agent errors
 . :question:
 
+=== How to generate a log into Jenkins manually for testing
+
+If you need to force a log to be sent to the backend, a possible hack is to open the `/script` URL, then paste and run the following code:
+
+[source,groovy]
+java.util.logging.Logger.getLogger('some.logger.name').warning('Warning log for testing');
```

Comment:
I believe this won't render properly, it needs to be "fenced" in `----` notation, e.g.

```asciidoc
[source,groovy]
----
java.util.Blah
----
```

Reply:
Nope. You are required to use `---` only when there are blank lines, then the parser will stop thinking this is the same source block at the first empty line.

Here is how it renders:

![image](https://user-images.githubusercontent.com/223853/40260588-bcbad8a6-5afb-11e8-8276-b1d575d6708a.png)